build.wake

```diff
@@ -32,6 +32,27 @@ def firrtlProto Unit =
   | runJob
   | getJobOutputs
 
+def firrtlVersion = "1.4.0-RC3"
+
+def buildInfo ver =
```

Comment:
Nit: Can you give `ver` a more descriptive name? There's two version properties in the generated Scala object, `version` and `scalaVersion`, and this one is actually the Scala version despite having a name that is more similar to the property that holds the package version.